docs/resources/kafka_connect_cluster_v2.md

```diff
@@ -0,0 +1,320 @@
+*___region___*<br>
+<ins>Type</ins>: string, required, immutable<br>
+<br>Region of the Data Centre. See the description for node size for a compatible Data Centre for a given node size.<br><br>
+*___name___*<br>
+<ins>Type</ins>: string, required, immutable<br>
+<br>A logical name for the data centre within a cluster. These names must be unique in the cluster.<br><br>
+*___node_size___*<br>
+<ins>Type</ins>: string, required, immutable<br>
+<br>Size of the nodes provisioned in the Data Centre. --AVAILABLE_NODE_SIZES_MARKER_V2_APACHE_KAFKA_CONNECT--<br><br>
+*___network___*<br>
+<ins>Type</ins>: string, required, immutable<br>
+<br>The private network address block for the Data Centre specified using CIDR address notation. The network must have a prefix length between `/12` and `/22` and must be part of a private address space.<br><br>
+### Input attributes - Optional
+*___azure_settings___*<br>
+<ins>Type</ins>: nested block, optional, immutable, see [azure_settings](#nested--azure_settings) for nested schema<br>
+<br>Azure specific settings for the Data Centre. Cannot be provided with AWS or GCP settings.<br><br>
+*___gcp_settings___*<br>
+<ins>Type</ins>: nested block, optional, immutable, see [gcp_settings](#nested--gcp_settings) for nested schema<br>
+<br>GCP specific settings for the Data Centre. Cannot be provided with AWS or Azure settings.<br><br>
+*___tag___*<br>
+<ins>Type</ins>: repeatable nested block, optional, immutable, see [tag](#nested--tag) for nested schema<br>
+<br>List of tags to apply to the Data Centre. Tags are metadata labels which  allow you to identify, categorize and filter clusters. This can be useful for grouping together clusters into applications, environments, or any category that you require.<br><br>
+*___aws_settings___*<br>
+<ins>Type</ins>: nested block, optional, immutable, see [aws_settings](#nested--aws_settings) for nested schema<br>
+<br>AWS specific settings for the Data Centre. Cannot be provided with GCP or Azure settings.<br><br>
+*___provider_account_name___*<br>
+<ins>Type</ins>: string, optional, immutable<br>
+<br>For customers running in their own account. Your provider account can be found on the Create Cluster page on the Instaclustr Console, or the "Provider Account" property on any existing cluster. For customers provisioning on Instaclustr's cloud provider accounts, this property may be omitted.<br><br>
+### Read-only attributes
+*___status___*<br>
+<ins>Type</ins>: string, read-only<br>
+<br>Status of the Data Centre.<br><br>
+*___id___*<br>
+<ins>Type</ins>: string, read-only<br>
+<br>ID of the Cluster Data Centre.<br><br>
+*___nodes___*<br>
+<ins>Type</ins>: repeatable nested block, read-only, see [nodes](#nested--nodes) for nested schema<br>
+<br>
+<a id="nested--custom_connectors"></a>
+## Nested schema for `custom_connectors`
+Defines the location for custom connector storage and access info.<br>
+### Input attributes - Optional
+*___azure_settings___*<br>
```

Comment:
Why do we have duplicated parameters like this?

Reply:
Not really, `awsSettings` here comes under `customConnectors`. The other one is under `dataCentre`, So the parent object here defines the context. The PD has the complete payload.

Reply:
Hey Yu, I have updated the names of these.